utils/zerovec/src/varzerovec/slice.rs

```diff
@@ -360,6 +360,28 @@ where
     }
 }
 
+impl<T> Eq for VarZeroSlice<T>
+where
+    T: VarULE,
+    T: ?Sized,
+    T: Eq,
+{
+}
+
+impl<T: VarULE + ?Sized + PartialOrd> PartialOrd for VarZeroSlice<T> {
```

Comment:
Question: Where do you use these impls?  If they aren't needed for this PR, consider moving them to their own PR

Reply:
They're needed so that we can `#[derive(Ord)]` etc as necessary